packages/documentation/src/pages/getting-started/index.mdx

```diff
@@ -3,38 +3,44 @@ title: Set up VA.gov locally
 ---
 
 # Set up VA.gov locally
+These instructions cover the installation of dependencies needed for running VA.gov locally.
+- **Node setup** - steps for installing `node` which is needed for building, testing, and running the local development server
+- **Get the source code** - steps for cloning the various repos needed to run VA.gov
+- **Start up the front end** - installing and running VA.gov locally
+- **Backend and local tools setup** - links to other setup instructions for running `vets-api` and retrieving static content
 
-## Node setup
+## Prerequisites
+- [`homebrew`](http://brew.sh/) is recommended for for installing `nvm` but other installation approaches are on [`nvm` Github page](https://github.com/creationix/nvm#installation-and-update)
 
-To get started, you'll need to install Node.js and Yarn. Versions 8.10.0 and 1.12.3, respectively.
+## Node setup
 
-The best way to do that is through a tool called `nvm`, which lets you manage multiple versions of Node on your machine.  This makes it easier to keep in sync with everyone else on the project. You can find installation instructions at [nvm's Github page](https://github.com/creationix/nvm#installation-and-update), but if you're on a Mac you can use [homebrew](http://brew.sh/) to install it:
+1. Install `nvm`:
 
 ```bash
 brew update && brew install nvm
 ```
-_There will be some further instructions in the success message after install "finishes"._
 
-Once you have `nvm` installed, you should install Node.js:
+_Follow the post install instructions in the success message._
+
+2. Install `node 8.10.0` (this also installs `npm`):
```

Comment:
I don't know if it's necessary to say this installs `npm` since we don't use it.